docs/wiki/The library - Module overview.md

```diff
@@ -0,0 +1,116 @@
+This section provides an overview of the library's feature set.
+
+---
+
+### _Navigation_
+
+- [Feature table](#feature-table)
+
+---
+
+# Feature table
+
+| # | Module | RBAC | Locks | Tags | Diag | EP | Pip-Depl. | # children | # lines |
```

Comment:
I'd suggest either to expand column names or better a legend to explain e.g. what EP means

Reply:
I did have bigger names, but Mate asked me to shorten them. A legend seems to be a good compromise - will add

Reply:
added